mantisshrimp/backbones/create_backbones.py

```diff
@@ -0,0 +1,33 @@
+__all__ = ["create_backbone_adaptive", "create_backbone_features"]
+
+from mantisshrimp.imports import *
+
+
+# Use this when adaptive avg_pooling is not available or you want to pass custom CNN
+# This can work with adaptive case too. So it is a bit generic case.
+def create_backbone_generic(model, out_channels: int):
```

Comment:
I believe this method might lead to mistakes, it's not really generic, it just removes the last layer from a model. If a have a model that is not structure in the way of `[backbone, head]` it will fail silently.

For example, my model can me like so: `[conv1, conv2, conv3, linear1, linear2, linear3]`, cutting the last layer would only remove `linear3`

Reply:
Yes, it will remove the last layer. But most CNN models are structured like this (for image classification) user might transfer his classification model into object detection model.

Reply:
Also refer the colab notebook. It explains what is happening.